src/app/routes/enrolment/view-requests/components/expiration-date/expiration-date.component.spec.ts

```diff
@@ -82,15 +82,16 @@ describe('ExpirationDateComponent', () => {
     );
   });
 
-  it('should check if remove button sets expiration time as undefined', () => {
+  it('should check if remove button calls "remove" to set the expiration time as null', () => {
     component.defaultValidityPeriod = 100;
     fixture.detectChanges();
 
     const { removeButton } = getSelectors(hostDebug);
     removeButton.click();
     fixture.detectChanges();
 
-    expect(component.add.emit).toHaveBeenCalledWith(undefined);
+    expect(component.remove.emit).toHaveBeenCalled;
```

Comment:
I think You need to add a spy for it like for `add`. And you lost brackets.
```suggestion
    expect(component.remove.emit).toHaveBeenCalled();
```